java/src/jmri/jmrit/roster/LocoFile.java

```diff
@@ -115,7 +115,7 @@ public static void loadCvModel(Element loco, CvTableModel cvModel, String mfgID,
                         cvObject = cvModel.allCvMap().get(name);
                      } else {
                         // this is a valid way to migrate a decoder definition, i.e. to remove a variable.
-                        log.info("CV {} was in loco file, but not defined by the decoder definition; migrated", name);
+                        log.info("CV {} was in '{}' roster file, but not defined by the decoder definition; migrated", name, rosterName);
```

Comment:
A suggestion:
I think it would be good to add the name of the decoder and manufacturer as well since it would help us in case anyone asks about this.

If a user asks on the JMRI users list, the roster name would not help us, but the decoder name would tell us which decoder it's about.

Reply:
I don't know why this happens.  I've never seen it.  

I'll add the extra information tomorrow.  We only have the manufacturer and family name here, not the exact model, but perhaps that's good enough.

Reply:
No access to the actual roster entry being processed?
It seems to be triggered NOT during startup up, so must be during roster entry open??

Reply:
> No access to the actual roster entry being processed?

Not sure what you're asking here.  `rosterName` is the name of the roster entry.  What else from the entry are you looking for? 

> It seems to be triggered NOT during startup up, so must be during roster entry open??

Correct.  Specifically, it's while reading the roster file for the locomotive to get the locomotive's current contents.

Reply:
The roster ENTRY name would be perfect for identifying which roster entry contains the “error”